vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomActivity.kt

```diff
@@ -113,7 +115,13 @@ class CreateDirectRoomActivity : SimpleFragmentActivity() {
     override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>, grantResults: IntArray) {
         if (allGranted(grantResults)) {
             if (requestCode == PERMISSION_REQUEST_CODE_READ_CONTACTS) {
-                addFragmentToBackstack(R.id.container, ContactsBookFragment::class.java)
+                try {
+                    addFragmentToBackstack(R.id.container, ContactsBookFragment::class.java)
```

Comment:
I wouldn't even try to do this, directly post it to postResume?

Reply:
the bug is only for Android 6, so I assume it is a workaround... But Ok I will change that

Reply:
It's more a question than a request, I don't know if it'll work